shell/browser/extensions/electron_component_extension_resource_manager.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
This file is copied from Chrome, then I removed a bunch of stuff that isn't relevant to Electron.